src/syscheckd/whodata/win_whodata.c

```diff
@@ -1040,6 +1233,10 @@ long unsigned int WINAPI state_checker(__attribute__((unused)) void *_void) {
         w_rwlock_unlock(&syscheck.wdata.directories->mutex);
 
         sleep(interval);
+
+        if (!policies_checked) {
```

Comment:
In this case it is necessary to wait a full interval to make sure to avoid the false positive that causes a restart of the Wazuh agent. 
Since the 4719 events of changes in the Audit Policies, caused by the restart, were being read. By waiting for a full interval we made sure to avoid them.